Bottom: fb0a8af95754419a92795a18dd158c5456e3000f
Top:    d53ddf7d9eba3864d0ba2baed126b459933e2a46
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-18 23:02:29 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 18113f1..2555d7a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -2887,10 +2887,7 @@ lesstif_notify_crosshair_change (bool changes_complete)
     invalidate_depth --;
 
   if (invalidate_depth < 0)
-    {
-      fprintf (stderr, "ERROR: Unmatched notify_crosshair_change calls\n");
-      invalidate_depth = 0;
-    }
+    invalidate_depth = 0;
 
   if (invalidate_depth == 0)
     DrawAttached ();
@@ -2908,10 +2905,7 @@ lesstif_notify_mark_change (bool changes_complete)
     invalidate_depth --;
 
   if (invalidate_depth < 0)
-    {
-      fprintf (stderr, "ERROR: Unmatched notify_mark_change calls\n");
-      invalidate_depth = 0;
-    }
+    invalidate_depth = 0;
 
   if (invalidate_depth == 0)
     DrawMark ();
